tests/conftest.py

```diff
@@ -280,7 +280,7 @@ def unicode_emojis():
     'reactions': [],
     'type': 'private',
     'avatar_url': 'dummy_avatar_url',
-    'subject_links': [],
+    'topic_links': [],
```

Comment:
We should check this in any case, as I'm not sure why a group-PM should have a topic? Same above.

Reply:
Yeah, I ran the tests without subject_links in pm_template and group_pm_template, and as expected, there was no error.

Maybe I'll mention this in the commit text as well. Thanks for pointing out :)